internal/resolution/resolver_test.go

```diff
@@ -19,112 +18,114 @@ import (
 )
 
 func TestOperatorResolver(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Operator Resolver Suite")
-}
 
-func FakeClient(objects ...client.Object) client.Client {
-	scheme := runtime.NewScheme()
-	if err := v1alpha1.AddToScheme(scheme); err != nil {
-		panic(fmt.Sprintf("error creating fake client: %s", err))
+	testEntityCache := map[deppy.Identifier]input.Entity{"operatorhub/prometheus/0.37.0": *input.NewEntity(
+		"operatorhub/prometheus/0.37.0", map[string]string{
+			"olm.bundle.path": `"quay.io/operatorhubio/prometheus@sha256:3e281e587de3d03011440685fc4fb782672beab044c1ebadc42788ce05a21c35"`,
+			"olm.channel":     "{\"channelName\":\"beta\",\"priority\":0,\"replaces\":\"prometheusoperator.0.32.0\"}",
+			"olm.gvk":         "[{\"group\":\"monitoring.coreos.com\",\"kind\":\"Alertmanager\",\"version\":\"v1\"}, {\"group\":\"monitoring.coreos.com\",\"kind\":\"Prometheus\",\"version\":\"v1\"}]",
```

Comment:
These aren't actually being used right now, right? It'll be cleaner to only include the properties that are actually being used (`olm.package`, `olm.bundle.path`), and include other fields like gvk iteratively as we start actually using them

Reply:
As long as FBC populates those properties, I think it's still useful to at least have them mocked on test objects.